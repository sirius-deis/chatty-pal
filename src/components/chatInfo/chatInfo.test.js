import { render, screen } from "@testing-library/react";
import ChatInfo from "./chatInfo";

describe("ChatInfo component", () => {
  it("should match snapshot", () => {
    const { container } = render(<ChatInfo name="some name" />);
    expect(container).toMatchSnapshot();
  });

  it("should render a name", () => {
    render(<ChatInfo name="some name" />);
    expect(screen.getByText(/some name/)).toBeInTheDocument();
  });
  it("should render a profile picture", () => {
    render(<ChatInfo name="Sam" photoSrc="picture.png" />);
    const img = screen.getByAltText(/Sam profile photo/);
    expect(img).toBeInTheDocument();
    expect(img).toHaveAttribute("src", "picture.png");
  });
});
