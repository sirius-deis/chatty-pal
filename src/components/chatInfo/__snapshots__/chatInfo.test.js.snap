// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChatInfo component should match snapshot 1`] = `
<div>
  <div
    class="sc-aXZVg euuOOg"
  >
    some name
    <img
      alt="some name profile photo"
    />
  </div>
</div>
`;
